FoodplannerDataAccessSql/Migrations/12_AddCascadeToUserId.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace FoodplannerDataAccessSql.Migrations;
 
-[Migration(11)]
+[Migration(12)]
```

Comment:
This migration should remain 11:
```suggestion
[Migration(11)]
```

Reply:
Nuh uh